sdk/ml/azure-ai-ml/tests/automl_job/unittests/test_automl_image_sweep_setting.py

```diff
@@ -0,0 +1,138 @@
+# ---------------------------------------------------------
```

Comment:
great to see this new file, I didn't like the duplication in the previous iteration :)